okio/jvm/jvm.gradle

```diff
@@ -1,13 +1,54 @@
+buildscript {
+  repositories {
+    maven {
+      url "https://plugins.gradle.org/m2/"
+    }
+  }
+  dependencies {
+    classpath deps.bnd
```

Comment:
Please add this to the root `build.gradle`. The buildscript classpath is global and it's much easier to deal with when there is only a single place where dependencies are added.

Reply:
For some reason gradle does not allow to access buildscript classes directly in a gradle file that is externally applied. Therefore as a workaround I have added the class as a property to the root project to make it accessible from the jvm.gradle file.

Reply:
Applying plugins in applied scripts works as intended but I don't want to apply the bnd plugin because on default the bnd plugin overwrites the jar plugin to provide the updated MANIFEST.MF and that could be counted as "intrusive" again.